articles/service-fabric/service-fabric-get-started-mac.md

```diff
@@ -64,34 +58,58 @@ To set up a local Docker container and have a Service Fabric cluster running on
     >
     >The recommended approach is to directly modify the daemon configuration settings in Docker. Select the **Docker icon**, and then select **Preferences** > **Daemon** > **Advanced**.
     >
+    >We recommend increasing the resources allocated to docker when testing large complex applications, this can be changed by selecting **Docker icon**, and then select **Advanced** then adjusting the number of cores and memory as required.
+
+2. In a new directory create a file called `.Dockerfile` to build your Service Fabric Image:
+
+    ```dockerfile
+    FROM servicefabricoss/service-fabric-onebox
+    WORKDIR /home/ClusterDeployer
+    RUN ./setup.sh
+    #Generate the local
+    RUN locale-gen en_US.UTF-8
+    #Set environment variables
+    ENV LANG=en_US.UTF-8
+    ENV LANGUAGE=en_US:en
+    ENV LC_ALL=en_US.UTF-8
+    EXPOSE 19080 19000 80 443
+    #Start SSH before running the cluster
+    CMD /etc/init.d/ssh start && ./run.sh
+    ```
 
-3. Start a Service Fabric onebox container instance and use the image that you pulled down in the first step:
+    >[!NOTE]
+    >You can adapt this file to add additional programs or dependencies into your container.
```

Comment:
This file can be modified to include your application dependencies. For example, .....